object_template.go

```diff
@@ -0,0 +1,86 @@
+package v8go
+
+// #include <stdlib.h>
+// #include "v8go.h"
+import "C"
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"runtime"
+	"unsafe"
+)
+
+// PropertyAttribute are the attributes for a property
```

Comment:
nit: no period.

Reply:
Hahah. The external docs are even less informative: https://v8.github.io/api/head/namespacev8.html#a05f25f935e108a1ea2d150e274602b87